### DISCUSSION TITLE
"Add card" button: add a possibility to move it

### DISCUSSION THREAD

**Discussion by ildar170975** [home-assistant/frontend#16999] [Dashboard]
Sometimes the "Add card" button overlaps controls for cards ("duplicate", "delete", ...).

![image](https://github.com/home-assistant/frontend/assets/71872483/33e408d6-b363-44bb-9865-68ea2664a700)

Not a big deal, a user may resize a window and make these controls visible.
But I wonder if it is possible to add a "grab & move" area to the card?
Smth like this:
![image](https://github.com/home-assistant/frontend/assets/71872483/a3440332-20da-447c-bb54-95326cc628aa)
The inner area is a "tap area", the "outer rim" is a "move area".

This is probably may only related to desktop browsers; for mobile browsers some another ways are needed...

**Comment by karwosts** [comment]
I'm not a CSS expert but it feels like the easiest thing may be to just add an invisible 150px padding element below all the card content in edit mode, so that you can just keep scrolling down a little further and the button doesn't overlap the card.

**Comment by karwosts** [comment]
Looks fixed by #17070. You still can't move the button, but it resolves the overlap issue.
